streaming/text/convert/c4.py

```diff
@@ -0,0 +1,109 @@
+import os
+from argparse import ArgumentParser, Namespace
+from typing import Any, Dict, Iterator
+
+import datasets
+from datasets.arrow_dataset import Dataset
+from torch.utils.data import DataLoader, IterableDataset, get_worker_info
+from tqdm import tqdm
+
+from ...base import MDSWriter
+
+
+def parse_args() -> Namespace:
+    """Parse commandline arguments.
+
+    Args:
+        Namespace: Commandline arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument('--out', type=str, default='/datasets/mds/c4-zstd/')
```

Comment:
Can we have the default datasets start from current directory rather than root `/`. For example, `./datasets/mds/c4-zstd/`. Sometimes, there could be a permission issue where the executor doesn't have a permission to create a directory in root.

Reply:
Sounds good to me. Also do you have a preference of using `--in` vs `--in_root` and `--out` vs `--out_root`. I prefer latter as it is more verbose.

Reply:
Sure, switching everyone to in_root/out_root